docs/2.8.0/docs/tools/assistant-build.rst

```diff
@@ -15,21 +15,502 @@ To compile your Daml source code into a Daml archive (a ``.dar`` file), run::
+
+.. code:: bash
+
+   > # ... make changes to Model ...
+   > cd ./package-Logic/ # Navigate to package-Logic
+   > daml build # Build package Logic
+   ...
+   Dependency "package-Model" is stale, rebuilding...
+   ...
+   Building "package-Logic"...
+   Done.
+
+With multi-package builds configured, you can run ``daml build`` just once to build interdependent packages, with a guarantee that changes to dependencies are always propagated wherever they are needed.
+
+Build all packages in a project
+-------------------------------
+
+To build all packages in a project, use the ``--all`` flag. This
+flag builds every package listed in the specified ``multi-package.yaml``. 
+With the ``--all`` flag, you can run ``daml build`` from outside a package directory.
+
+.. _multi-package-yaml-location:
+
+How the CLI finds multi-package.yaml
+------------------------------------
+
+In most cases, you'll run ``daml build`` and ``daml clean`` from deeper in your project structure than the root. The discovery logic for these commands follows the patterns of ``daml.yaml``:
+
+1. If a path is specified with ``--multi-package-path PATH``, use 
+   the ``multi-package.yaml`` at that location. (This is the
+   ``multi-package`` equivalent to ``--project-root``.)
+2. Otherwise, search up the directory tree starting from the directory where  
+   either ``daml build`` or ``daml clean`` was invoked. Return the first 
+   ``multi-package.yaml`` encountered.
+3. If no ``multi-package.yaml`` file is found in the preceding steps, do 
+   not use the multi-build feature.
+
+Add a package to a multi-package configuration
+----------------------------------------------
+
+In the example above, the ``multi-package.yaml`` file registers two packages,
+Logic and Model:
+
+::
+
+   > tree
+   .
+   ├── multi-package.yaml
+   ├── package-Logic
+   │   ├── .daml/dist/package-Logic-1.0.0.dar
+   │   ├── daml/...
+   │   └── daml.yaml
+   └── package-Model
+       ├── daml/...
+       ├── daml/.dist/package-Model-1.0.0.dar
+       └── daml.yaml
+
+To add a new package called Tests to this example structure, run 
+``daml new --template=empty-skeleton package-Tests``. This command creates an
+empty package named ``package-Tests``, parallel to the existing
+``package-Logic`` and ``package-Model`` packages.
+
+::
+
+   > tree
+   .
+   ├── multi-package.yaml
+   ├── package-Logic
+   ├── package-Model
+   └── package-Tests
+
+In the newly created ``package-Tests/daml.yaml``, remove the
+``sandbox-options`` and add a dependency on the DAR for ``Logic``:
+
+.. code:: bash
+
+   data-dependencies:
+   - ../package-Logic/.daml/dist/package-Logic-1.0.0.dar
+
+Finally, add ``package-Testing`` to the ``multi-package.yaml`` file:
+
+``multi-package.yaml``
+
+.. code:: diff
+
+     packages:
+     - ./package-Logic
+     - ./package-Model
+   + - ./package-Tests
+
+Run multi-package tests
```

Comment:
This header is smaller than the main text in the live-preview, we might want to change this

Reply:
This is a bigger and ongoing issue that we'll have to address in a theme/css fix later.